specs/PermissionManagement.md

```diff
@@ -0,0 +1,554 @@
+Permission Management APIs
+===
+
+# Background
+
+The WebView2 team is adding support for more permission management scenarios. With the
+extended `PermissionRequested` event args and new permission APIs you will be able to:
+- Set permission for a site at any time
+- List all nondefault permissions
+- Control more permission kinds
+
+The new `ShouldPersist` property on the `PermissionRequestedEventArgs` lets you
```

Comment:
Yes ShouldPersist guarantees the permission will be remembered underneath the app - this is the current behavior for the event. We want to allow developers to turn off that behavior with a new property, but keep the current behavior as the default to avoid breaking apps